### PR TITLE
phython3-{kconfiglib, jinja2, markupsafe}: add new packages

### PR DESCRIPTION
**Pull request by blogic**
Maintainer: me / @blogic 
Compile tested: arm/64, x86
Run tested: arm/64, x86

Description:
Add 3 new python packages,
python3-kconfiglib: add apckage
python3-jinja2: add package
python3-markupsafe: add package

### CODE REVIEW

File: lang/python/python3-jinja2/Makefile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-jinja2
```
Comment (BKPepe) [inline lang/python/python3-jinja2/Makefile:3]: Why? This is already here:
https://github.com/openwrt/packages/blob/master/lang/python/Jinja2/Makefile

File: lang/python/python3-markupsafe/Makefile [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=MarkupSafe
```
Comment (BKPepe) [inline lang/python/python3-markupsafe/Makefile:7]: https://github.com/openwrt/packages/blob/master/lang/python/MarkupSafe/Makefile

**Review by BKPepe — CHANGES_REQUESTED** [review]
Two packages, which are you adding are already here.

File: lang/python/python3-kconfiglib/Makefile [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+PKG_RELEASE:=1
+PKG_VERSION:=13.1.1
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/ulfalizer/Kconfiglib.git
+PKG_SOURCE_VERSION:=bf01114a0ce09b891ed04609ea7e9cc6f3ef3df2
+PKG_BUILD_DIR:=$(BUILD_DIR)/python3-kconfiglib-$(PKG_VERSION)
+PKG_LICENSE:=ISC
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=John Crispin <john@phrozen.org>
+HOST_BUILD_DEPENDS:=python3/host
```
Comment (BKPepe) [inline lang/python/python3-kconfiglib/Makefile:13]: This looks bad. It's ugly. Please take a look at how other packages in the Python folder looks like.
E.g. https://github.com/openwrt/packages/blob/master/lang/python/python-attrs/Makefile

Comment (blogic) [reply lang/python/python3-kconfiglib/Makefile:13]: @BKPepe let me rephrase that for you "Hi kindly consider to making this patch aligned with how other packages do it. take a look at <...> for a referecne"

File: lang/python/python3-kconfiglib/Makefile [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	URL:=https://github.com/ulfalizer/Kconfiglib
+endef
+
+define Package/python3-kconfiglib/description
+	Kconfiglib is a Kconfig implementation in Python 2/3.
+endef
+
+define Host/Compile
+	$(call HostPython3/ModSetup,,install --prefix="" --root="$(STAGING_DIR_HOSTPKG)")
+endef
```
Comment (jefferyto) [inline lang/python/python3-kconfiglib/Makefile:34]: If you have another Python package that needs this host package, it may be easier to add a [host-pip-requirements](https://github.com/openwrt/packages/tree/master/lang/python/host-pip-requirements) file, then set [`HOST_PYTHON3_PACKAGE_BUILD_DEPENDS`](https://github.com/openwrt/packages/tree/master/lang/python#host-side-python-packages-for-build).

**Comment by commodo** [comment]
hey

is there any interest in this now?
AFAICT, 2 packages are already in the feed;
`kconfiglib` remains to be discussed;

thoughts? plans?

**Comment by commodo** [comment]
Closing this.
We can re-open / add `kconfiglib` later if needed.

But this has stalled a bit.
